waffle/__init__.py

```diff
@@ -7,6 +7,7 @@
 
 VERSION = (0, 13, 0)
```

Comment:
This is also something I noticed. You might want to fix this! 😬

Reply:
Ah, I see someone else [beat me to it](https://github.com/django-waffle/django-waffle/pull/323)!